crates/biome_lsp/src/session.rs

```diff
@@ -709,7 +719,13 @@ impl Session {
 
         self.insert_and_scan_project(project_key, path.into());
 
-        if let Err(error) = result {
+        if let Err(WorkspaceError::PluginErrors(error)) = result {
+            error!("Failed to load plugins: {:?}", error);
+            self.client
+                .log_message(MessageType::ERROR, format!("{:?}", error))
```

Comment:
Nit: identifiers can be inlined in format strings.

```suggestion
            error!("Failed to load plugins: {error:?}");
            self.client
                .log_message(MessageType::ERROR, format!("{error:?}"))
```